esmvaltool/utils/cmorizers/obs/cmorize_obs_CRU.py

```diff
@@ -0,0 +1,66 @@
+"""ESMValTool CMORizer for CRU data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://crudata.uea.ac.uk/cru/data/hrg/cru_ts_4.02/cruts.1811131722.v4.02/
+
+Last access
+    20190516
+
+Download and processing instructions
+    Download the following files:
+        cru_ts4.02.1901.2017.{raw_name}.dat.nc
```

Comment:
Actually `{raw_name}/cru_ts4.02.1901.2017.{raw_name}.dat.nc`

Reply:
The files are available as .gz, which have to be unzipped before running the cmorizer.
Please add this information to the processing instructions.

Reply:
Or add code to do it?

Reply:
Yes, but saving the output in the output_dir, since the user does not necessarily has writing rights to rawobs_dir.

Reply:
> Actually `{raw_name}/cru_ts4.02.1901.2017.{raw_name}.dat.nc.gz`

I saved only the files which are need manually (there are many more files which are not needed), so in this case `cru_ts4.02.1901.2017.{raw_name}.dat.nc.gz` was enough. If you want, I can change it. How did you download your files? I didn't get `wget` to download only the necessary files recursively.

 > Or add code to do it?

I will try that.

Reply:
`wget https://crudata.uea.ac.uk/cru/data/hrg/cru_ts_4.02/cruts.1811131722.v4.02/pre/cru_ts4.02.1901.2017.pre.dat.nc.gz`
and
`wget https://crudata.uea.ac.uk/cru/data/hrg/cru_ts_4.02/cruts.1811131722.v4.02/tmp/cru_ts4.02.1901.2017.tmp.dat.nc.gz`
worked for me (at DKRZ).

So, given the source, the file(s) `{raw_name}/cru_ts4.02.1901.2017.{raw_name}.dat.nc.gz` should be downloaded.

Reply:
Okay, I will change that :+1:  (I used the `-nd` option of `wget` which saves everything in the same directory).

Reply:
Oh, I think i misunderstood @mattiarighi. You only want me to change the description, not the code itself, right? After your command, my `Tier2` directory looks like this

```
CRU
├── cru_ts4.02.1901.2017.tmp.dat.nc
...
```

so there's no leading `tmp/` directory.

Reply:
Yes, exactly, just the description. :+1: